runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/users/register/view.ts

```diff
@@ -0,0 +1,11 @@
+const viewRegisterTranslations = {
+    'Users.Register.View.Events.Success.AcceptingUser': 'Success',
+    'Users.Register.View.Events.Error.RolesNotSelected': 'Not selected all required roles',
+    'Users.Register.View.Events.Error.AcceptFailed': 'Could not accept selected users',
+    'Users.Register.View.Events.Error.FindingUsers': 'Error finding not activated users',
```

Comment:
is it ERROR (e.g. we get 404/403/501) or is it MESSAGE (that we search but, there are no users in DB)?

if it's a MESSAGE rename translate key and translation to "Could not find activated users"